scripts/init.sh

```diff
@@ -88,6 +88,39 @@ if ! [ -f "${ENV_FILE}" ]; then
   SCNODE_NET_NODENAME="ext-${role_value}-$((RANDOM % 100000 + 1))" || fn_die "Error: could not set NODE_NAME variable for some reason. Fix it before proceeding any further.  Exiting..."
   sed -i "s/SCNODE_NET_NODENAME=.*/SCNODE_NET_NODENAME=${SCNODE_NET_NODENAME}/g" "${ENV_FILE}"
 
+  # Setting explicit wallet address for forger rewards to be sent to vs default local forger
+  if [ "${role_value}" = "forger" ] && [ "${network_value}" = "eon" ]; then
+    echo -e "\n\033[1m=== Setting up FORGER rewards address ===\033[0m\n"
+    read -rp "Do you want to provide the wallet address for your FORGER rewards to be sent to(FORGER local wallet address is used by default)? ('yes' or 'no') " reward_address_set
+    while [[ ! "${reward_address_set}" =~ ^(yes|no)$ ]]; do
+      echo -e "\nWarning: The only allowed answers are 'yes' or 'no'. Please try again...\n"
+      read -rp "Do you want to provide the wallet address for your FORGER rewards to be sent to(FORGER local wallet address is used by default)? ('yes' or 'no') " reward_address_set
+    done
+    if [ "${reward_address_set}" = "yes" ]; then
+      read -rp "Please type or copy/paste here the wallet address for your FORGER rewards to be sent to: " forger_reward_address
+      echo -e "\nYou have provided the following wallet address: \033[1m${forger_reward_address}\033[0m\n"
+      read -rp "Do you confirm this is the wallet address you want to use for your FORGER rewards to be sent to: ${forger_reward_address}? ('yes' or 'no') " reward_address_set_2
+      while [[ ! "${reward_address_set_2}" =~ ^(yes|no)$ ]]; do
+        echo -e "\nWarning: The only allowed answers are 'yes' or 'no'. Please try again...\n"
+        read -rp "Do you confirm this is the wallet address you want to use for your FORGER rewards to be sent to: ${forger_reward_address}? ('yes' or 'no') " reward_address_set_2
+      done
+      if [ "${reward_address_set_2}" = "yes" ]; then
+        forger_reward_address="$(strip_0x "${forger_reward_address}")"
+        if [ "${forger_reward_address}" != "invalid" ]; then
+          sed -i "s/SCNODE_FORGER_REWARD_ADDRESS=.*/SCNODE_FORGER_REWARD_ADDRESS=${forger_reward_address}/g" "${ENV_FILE}"
+        else
+          # Removing env_file since exit script will leave it in incomplete state
+          rm -f "${ENV_FILE}"
+          fn_die "Error: Provided wallet address is in the WRONG format!!! Please re-run the init.sh script again. Exiting ..."
```

Comment:
It would be useful to provide a format example, to help them identifying what's wrong?